### ISSUE TITLE
function to support multipage app

### ISSUE THREAD

**Issue by agaitan026** [openblocks-dev/openblocks#64]
Hi just like other lowcode and nocode app they have option to manage several pages inside same app, not like retool that works all with 'apps'

**Comment by sarike** [comment]
<img width="220" alt="image" src="https://user-images.githubusercontent.com/1788906/203890425-73e22925-31db-46fb-9839-f841e3a19ce8.png">

try this

**Comment by agaitan026** [comment]
nice, works, but still diff apps :( not pages

**Comment by neon-balcony** [comment]
@agaitan026 Yes, in Openblocks we tend to refer a single functioning page as an _app_, and you can use **navigation** to organize multi apps/pages.
